cloudinstall/machinewait.py

```diff
@@ -124,6 +124,7 @@ def update(self):
         # ensure that the button is always focused:
         self.main_pile.focus_position = len(self.main_pile.contents) - 1
 
+    # XXX: Is this utils.async needed?
```

Comment:
so i think the way to do this now should be to still have `do_continue` here, but instead of decorating it with utils.async, just have it look like this:

```python
def do_continue(self, *args, **kwargs):
    async.pool.submit(self.installer.do_install)
```